pyproject.toml

```diff
@@ -17,6 +17,10 @@ readme = "README.md"
 dynamic = ["version"]
 
 dependencies = [
+    'h5py',
+    'imageio_ffmpeg',
+    'numpy',
+    'pytest'
```

Comment:
You may want to move pytest in the dev dependencies list. When a user runs `pip install aind-ophys-utils` it will only install the packages needed for running the core code and leave out the packages the devs need to run tests.